examples/inter_process_ping_pong.py

```diff
@@ -66,32 +59,29 @@ def run_proc2():
 
 
 async def proc2_worker():
-    async with AsyncioEndpoint.serve(ConnectionConfig.from_name("e2")) as endpoint:
-        await endpoint.connect_to_endpoints(ConnectionConfig.from_name("e1"))
-        asyncio.ensure_future(display_proc1_events(endpoint))
-        endpoint.subscribe(
+    config = ConnectionConfig.from_name("e1")
+    async with AsyncioEndpoint("e2").run() as client:
+        await client.connect_to_endpoints(config)
+        asyncio.ensure_future(display_proc1_events(client))
+        client.subscribe(
             FirstThingHappened,
             lambda event: logging.info(
                 "Received via SUBSCRIBE API in proc2: %s", event.payload
             ),
         )
+        await client.wait_until_any_remote_subscribed_to(SecondThingHappened)
+
         while True:
             logging.info("Hello from proc2")
-            if is_nth_second(2):
-                await endpoint.broadcast(SecondThingHappened("Hit from proc2 "))
-            await asyncio.sleep(1)
+            await client.broadcast(SecondThingHappened("Hit from proc2 "))
+            await asyncio.sleep(2)
 
 
 async def display_proc1_events(endpoint):
     async for event in endpoint.stream(FirstThingHappened):
         logging.info("Received via STREAM API in proc2: %s", event.payload)
 
 
-# Helper function to send events every n seconds
-def is_nth_second(interval):
-    return int(time.time()) % interval is 0
```

Comment:
This didn't appear to be necessary since we can accomplish roughly the same thing using `await asyncio.sleep(interval)`